net-im/libcommuni/libcommuni-3.5.0_p20190110.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+EGIT_COMMIT="88bb0fcdaab5bdd1176a3b6b485a6415a137ec27"
+
+inherit qmake-utils
+
+DESCRIPTION="A cross-platform IRC framework written with Qt"
+HOMEPAGE="http://communi.github.io/"
+SRC_URI="https://github.com/communi/${PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="qml test +uchardet"
+
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtdeclarative:5
+	uchardet? ( app-i18n/uchardet )
+	!uchardet? ( dev-libs/icu:= )
+	"
```

Comment:
Mis-indent (doesn't this show as red in vim?)

Reply:
Nope, doesn't show as red here. Thanks, fixed.